src/package/Microsoft.NET.Test.Sdk/Microsoft.NET.Test.Sdk.targets

```diff
@@ -0,0 +1,6 @@
+<Project InitialTargets="NonCompatibleTargetFrameworkError_Microsoft_NET_Test_Sdk_net8_0">
+  <Target Name="NonCompatibleTargetFrameworkError_Microsoft_NET_Test_Sdk_net8_0"
+          Condition="'$(SuppressTfmSupportBuildWarnings)' == ''">
+    <Warning Text="Microsoft.NET.Test.Sdk doesn't support $(TargetFramework) and has not been tested with it. Consider upgrading your TargetFramework to net8.0 or later. You may also set &lt;SuppressTfmSupportBuildWarnings&gt;true&lt;/SuppressTfmSupportBuildWarnings&gt; in the project file to ignore this warning and attempt to run in this unsupported configuration at your own risk." />
```

Comment:
```suggestion
    <Warning Text="Microsoft.NET.Test.Sdk doesn't support $(TargetFramework) and has not been tested with it. Consider upgrading your TargetFramework to net8.0 or later, or net462 or later. You may also set &lt;SuppressTfmSupportBuildWarnings&gt;true&lt;/SuppressTfmSupportBuildWarnings&gt; in the project file to ignore this warning and attempt to run in this unsupported configuration at your own risk." />
```

Reply:
this text is what dotnet/runtime packages use for their netstandard compatibility warnings., so I would like to stick with that.

Reply:
@nohwnd As far as I understand now, with specific TFMs in builtMultiTargeting, you get warnings from NuGet, but the wording isn't always clear, so we plug our own warning (well, copied from dotnet/runtime 😄 ). Is that correct?

Reply:
Yes, (same screen as above):

![image](https://github.com/user-attachments/assets/e72111ac-26bc-4a8e-a6e3-a6f71347fdf8)

This is what you see when you try to restore. No exact description, vague language saying that we might not be compatible, and no info about how to suppress if you think you are doing it right.